sympy/functions/combinatorial/numbers.py

```diff
@@ -115,6 +115,9 @@ def eval(cls, n, sym=None):
     def _eval_rewrite_as_sqrt(self, n):
         return 2**(-n)*sqrt(5)*((1 + sqrt(5))**n - (-sqrt(5) + 1)**n) / 5
 
+    def _eval_rewrite_as_GoldenRatio(self,n):
+        return (S.GoldenRatio**n - (-1 * S.GoldenRatio)**(-1*n))/(2*S.GoldenRatio-1)
```

Comment:
It is probably more efficient to write `-S.GoldenRatio` instead of `-1 * S.GoldenRatio`. Maybe also `1/(-S.GoldenRatio)**n` tp avoid -1 in the exponent.